plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1178,6 +1178,8 @@ blockly.block.literal=Literal parameter %1 %2 %3
 blockly.block.literal.tooltip=A built-in option is something like ''day'', ''noon'', ''normal'', ''easy''.
 blockly.block.entity_get_armor_slot_item.tooltip=ID 3 is helmet, ID 0 are boots
 blockly.block.entity_set_armor_slot_item.tooltip=ID 3 is helmet, ID 0 are boots
+blockly.block.entity_get_ticks_frozen=Get number of ticks %1 has been frozen for
+blockly.block.entity_set_ticks_frozen=Set number of ticks %1 has been frozen for to %2
```

Comment:
Isn't this more `Freeze %2 for %1 ticks`

Reply:
what?

Reply:
You are right, before it was actually more correct on what it does, I will evert it. Sorry